.github/workflows/update-extensions.yml

```
@@ -4,44 +4,55 @@ on:
-          git config --global user.email "vbotbuildovich@users.noreply.github.com"
-          git add package.json package-lock.json
-          git commit -m "auto-docs: Update @redpanda-data/docs-extensions-and-macros"
-          git push origin ${{ matrix.branch }}
-        env:
-          ACCESS_TOKEN: ${{ env.ACTIONS_BOT_TOKEN }}
+
+      - name: Create pull request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: ${{ env.ACTIONS_BOT_TOKEN }}
+          commit-message: auto-docs: Update @redpanda-data/docs-extensions-and-macros
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 52: mapping values are not allowed in this context

(syntax-check)